src/vaadin-crud.html

```diff
@@ -60,8 +60,8 @@
       }
     </style>
 
-    <div id="container">
-      <div id="main">
+    <div id="container" part="container">
+      <div id="main" part="main">
```

Comment:
I am not fully happy with the names, but couldn't find better ones.

Reply:
Not sure, but I guess there is no need in exposing `main` in that case as you most probably will get access to it via `[part="container"] > div`.

Reply:
Well, true, I thought it would more safe to expose more parts, but if `[part="container] > div` is ok, then it may be fine.

Reply:
`[part="container"] > div` is an unsupported selector, so we should not encourage anyone to use those.

I don’t see any semantic purpose for exposing those elements for styling. They look very much just like internal layout implementation details, which the developer should have no interest in styling separately.